src/util.rs

```diff
@@ -1,28 +1,44 @@
 use std::fs::File;
 use std::io::{Read, Write};
 use std::path::Path;
-use std::{fs, io};
+use std::fs;
 
 use toml::{Parser, Value};
+use walkdir::WalkDir;
 
 use errors::*;
 
 pub fn cp_r(src: &Path, dst: &Path) -> Result<()> {
-    (|| -> io::Result<()> {
-            for e in src.read_dir()? {
-                let e = e?;
-                let f = e.file_name();
+    for e in WalkDir::new(src) {
+        let e = e.unwrap();
```

Comment:
Hmm, shouldn't this error be lifted? I'm not sure about the exact failure mode here. Maybe it happens if the directory gets removed while walking it?